src/journal/JournalPlayer.cc

```diff
@@ -44,16 +46,89 @@ struct C_HandleEntriesAvailable : public Context {
   }
 };
 
+struct PriCache : public PriorityCache::PriCache {
+  std::string m_name;
+  uint64_t m_max_size;
+
+  int64_t m_cache_bytes = 0;
+  int64_t m_committed_bytes = 0;
+  double m_cache_ratio = 0;
+
+  PriCache(const std::string &name, uint64_t max_size)
+    : m_name(name), m_max_size(max_size) {
+  }
+
+  int64_t request_cache_bytes(PriorityCache::Priority pri,
+                              uint64_t total_cache) const override {
+    if (pri != PriorityCache::Priority::PRI0) {
```

Comment:
Nit: perhaps just add ```ceph_assert(pri == PriorityCache::Priority::PRI0)``` since it seems like the other priorities aren't used / supported.